ToyBox/classes/MonkeyPatchin/BagOfPatches/MiscWrath.cs

```diff
@@ -675,6 +677,29 @@ private static void Postfix(Polymorph __instance) {
                 __instance.Owner.View.transform.localScale = new Vector3(scale, scale, scale);
             }
         }
+        [HarmonyPatch(typeof(LocalizedString))]
+        internal class BPTagger 
+        {
+            private static LocalizationPack pack = Kingmaker.Localization.LocalizationManager.LoadPack(Kingmaker.Localization.LocalizationManager.s_CurrentLocale);
```

Comment:
Any specific reason for not using `LocalizationManager.CurrentPack`?

If you can indeed use LocalizationManager.CurrentPack, then I'd also recommend switching to a getter (=>) instead of assignment. If you're loading the current pack for a specific reason then assignment is still preferable to prevent reloading on every LoadString call

Reply:
Did the patch you suggested in Discord (Well assuming I read your comment correctly)

LoadPack shouldn't be getting called every LoadString, it's outside the method